File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/OperationOutcome.ts
import { CodeableConcept } from "./CodeableConcept";
import { DomainResource } from "./DomainResource";
import { BackboneElement } from "./BackboneElement";

export type OperationOutcomeIssue = BackboneElement & {
    code: string;
    details?: CodeableConcept;
    location?: string[];
    severity: string;
    expression?: string[];
    diagnostics?: string;
};

export type OperationOutcome = DomainResource & {
    issue: OperationOutcomeIssue[];
};

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/OperationOutcome.py
from typing import Optional, List
from base import CodeableConcept
from base import DomainResource
from base import BackboneElement

class OperationOutcome_Issue(BackboneElement):
    code: str
    details: Optional[CodeableConcept] = None
    location: Optional[List[str]] = None
    severity: str
    expression: Optional[List[str]] = None
    diagnostics: Optional[str] = None

class OperationOutcome(DomainResource):
    issue: list[OperationOutcome_Issue] = []